core-plugins/src/e2e-test/features/groupby/GroupByWithFile.feature

```diff
@@ -0,0 +1,154 @@
+
+  @FILE_SOURCE_TEST @FILE_SINK_TEST
+  Scenario: To verify data is getting transferred from File to File sink plugin successfully with GroupBy plugin
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "File" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Analytics"
+    When Select plugin: "Group By" from the plugins list as: "Analytics"
+    Then Connect plugins: "File" and "Group By" to establish connection
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "Group By" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "groupByGcsCsvFile"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "groupByCsvDataTypeFileSchema"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "Group By"
+    Then Enter the GroupBy plugin fields as list "groupByValidFields"
+    Then Enter GroupBy plugin Fields to be Aggregate "groupByGcsAggregateFields"
+    Then Click on the Get Schema button
+    Then Click on the Validate button
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "File2"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "filePluginOutputFolder"
+    Then Replace input plugin property: "pathSuffix" with value: "yyyy-MM-dd-HH-mm-ss"
+    Then Select dropdown plugin property: "format" with option value: "tsv"
+    Then Click plugin property: "writeHeader"
+    Then Validate "File2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count of groupby is equal to IN record count of sink
+#    Enable below step once issue fixed (PLUGIN #1326)
+#    Then Validate output file generated by file sink plugin is equal to expected output file {string}
```

Comment:
We need to investigate why a large number of partitions are created - this will be done as part of PLUGIN-1326. However, your validation logic should remain the same - read contents of all the output files (part-r-xxx) and verify that the data written matches what you expect. Can you update the tests to do this?

Reply:
Done.